scripts/ci/ci-stateless-tests-cluster.sh

```diff
@@ -3,7 +3,7 @@
 # SPDX-License-Identifier: Apache-2.0.
 
 echo "Starting Standalone fuse-query"
-./scripts/deploy/fusequery-standalone.sh
+./scripts/deploy/fusequery-cluster-3-nodes.sh
```

Comment:
:+1:

Reply:
In order to make the cluster test running for other patches, I have fixed it in https://github.com/datafuselabs/datafuse/pull/607

Reply:
👍  Merged master.